jetstream/whats-new-notification-sidebarvertical-tabs.toml

```diff
@@ -0,0 +1,34 @@
+
+[experiment]
+
+#segment information:https://docs.telemetry.mozilla.org/concepts/segments.html
+segments = ["clients_last_seen.is_new_or_resurrected_v3", "clients_last_seen.activity_segments_v1"]
+
+
+[metrics]
+# Metrics to compute for each weekly analysis window.
+weekly = ["notification_clicks","avg_max_concurrent_vertical_tab_count"]
+
+# Metrics to compute for each 28 days long analysis window
+28_day = ["notification_clicks","avg_max_concurrent_vertical_tab_count"]
+
+# Metrics to compute only for the overall analysis window.
+overall = ["notification_clicks","avg_max_concurrent_vertical_tab_count"]
+
+[metrics.avg_max_concurrent_vertical_tab_count]
```

Comment:
I don't think the average number is the right metric for this experiment, I'd go with a binomial yes/no for having at least one at some point (i.e. trying vertical tabs)

Reply:
+ 1

The brief isn't super clear on what they are looking for in terms of engagement, but a 0/1 will be less noisy than an avg # of concurrent vertical tab count.

It would be something like this:

`CAST(SUM(COALESCE(payload.processes.parent.scalars.browser_engagement_max_concurrent_vertical_tab_count ,0)) > 0 AS INT)`

I note to use SUM() because this is coming from the `main` table which can have multiple rows per client in a day.